pkg/dp-server/server/server.go

```diff
@@ -80,9 +80,10 @@ func (d *DpServer) Start(stop <-chan struct{}) error {
 		return err
 	}
 	server := &http.Server{
-		Addr:      fmt.Sprintf(":%d", d.config.Port),
-		Handler:   http.HandlerFunc(d.handle),
-		TLSConfig: tlsConfig,
+		ReadHeaderTimeout: time.Second,
```

Comment:
Any particular reason to we should be setting only `ReadHeaderTimeout` and not `ReadTimeout`, the value of which is used for `ReadHeaderTimeout`?

Reply:
ReadHeadTimeout is easy to set (1second is forever) for a body timeout it's trickier. What if we're sending large payload. ReadHeaderTimeout is simpler to set: https://medium.com/a-journey-with-go/go-understand-and-mitigate-slowloris-attack-711c1b1403f6

I guess we could look at setting ReadTimeout in some palces

Reply:
That's a good reason!